src/main.js

```diff
@@ -73,8 +73,8 @@ const generateOneProp = (propType, propName) => {
   }
 }
 
-const forceGenerateOneProp = (propType) => {
-  const generate = GENERATORS[propType.type]
+const forceGenerateOneProp = (propType, propName) => {
+  const generate = GENERATORS[propType.type].bind(this, propName, !propType.isRequired)
```

Comment:
What function is the `isRequired` argument serving here?

Reply:
This was out of novelty, I figured there might be a use to getting this information to the user. In light of the changes that you discussed in your previous comment, I am going to strip this out so that this PR has less overhead. If you decide it would be useful in the future then it can easily be added back in.

Reply:
Ahhhhh okay, makes sense. Thank you for the explanation, if we find a good usecase for it in the future I'm open to the idea 👍